.github/dependabot.yml

```diff
@@ -25,12 +25,3 @@ updates:
       - "gradle"
     commit-message:
       prefix: "[WOR-1448]"
-    ignore:
```

Comment:
I am recalling now that this may have been related to running integration tests locally (a version incompatibility that we oddly didn't see in ci). I have checked out your branch and am running the tests.

Reply:
Thanks for the historical context and trying out these local tests!  But wanted to double-check, did you mean to comment this on the `.github/dependabot.yml` change?

Reply:
Yes, I recalled it being the reason we stopped automatic updates. But the integration tests did all pass for me localy.

Reply:
Thanks very much for checking!